packages/blade/src/components/Charts/AreaChart/AreaChart.stories.tsx

```diff
@@ -0,0 +1,328 @@
+    </StoryPageWrapper>
+  );
+};
+
+export default {
+  title: 'Components/Charts/AreaChart',
+  component: ChartArea,
+  tags: ['autodocs'],
+  argTypes: {
+    ...getStyledPropsArgTypes(),
+  },
+  parameters: {
+    docs: {
+      page: Page,
+    },
+  },
+} as Meta<typeof ChartArea>;
+
+// Sample data for charts
+const chartData = [
+  { month: 'Jan', teamA: 4000, teamB: 2400, teamC: 1800 },
+  { month: 'Feb', teamA: 3000, teamB: 1398, teamC: 2200 },
+  { month: 'Mar', teamA: 2000, teamB: 9800, teamC: 1500 },
+  { month: 'Apr', teamA: 2780, teamB: 3908, teamC: 2800 },
+  { month: 'May', teamA: 1890, teamB: 4800, teamC: 2100 },
+  { month: 'Jun', teamA: 2390, teamB: 3800, teamC: 2500 },
+];
+
+const data = [
+  {
+    name: 'Page A',
+    uv: 4000,
+    pv: 2400,
+    amt: 2400,
+  },
+  {
+    name: 'Page B',
+    uv: 3000,
+    pv: 1398,
+    amt: 2210,
+  },
+  {
+    name: 'Page C',
+    uv: 2000,
+    pv: 9800,
+    amt: 2290,
+  },
+  {
+    name: 'Page D',
+    pv: 3908,
+    amt: 2000,
+    uv: null,
+  },
+  {
+    name: 'Page E',
+    uv: 1890,
+    pv: 4800,
+    amt: 2181,
+  },
+  {
+    name: 'Page F',
+    uv: 2390,
+    pv: 3800,
+    amt: 2500,
+  },
+  {
+    name: 'Page G',
+    uv: 3490,
+    pv: 4300,
+    amt: 2100,
+  },
+];
+
+// 2.2.a - Simple Area Chart
+export const SimpleAreaChart: StoryFn<typeof ChartArea> = () => {
+  return (
+    <Box width="100%" height="500px">
+      <ChartAreaWrapper data={chartData}>
+        <ChartCartesianGrid />
+        <ChartXAxis dataKey="month" />
+        <ChartYAxis />
+        <ChartTooltip />
+        <ChartArea
+          dataKey="teamA"
+          name="Team A"
+          type="monotone"
+          color="chart.background.categorical.azure.intense"
+        />
+      </ChartAreaWrapper>
+    </Box>
+  );
+};
+
+// 2.2.b - Stacked Area Chart
+export const StackedAreaChart: StoryFn<typeof ChartArea> = () => {
+  return (
+    <Box width="100%" height="500px">
+      <ChartAreaWrapper data={chartData}>
+        <ChartCartesianGrid />
+        <ChartXAxis dataKey="month" />
+        <ChartYAxis />
+        <ChartTooltip />
+        <ChartLegend />
+        <ChartArea
+          dataKey="teamA"
+          name="Team A"
+          type="monotone"
+          stackId="1"
+          color="chart.background.categorical.azure.moderate"
+        />
+        <ChartArea
+          dataKey="teamB"
+          name="Team B"
+          type="monotone"
+          stackId="1"
+          color="chart.background.categorical.emerald.moderate"
+        />
+      </ChartAreaWrapper>
+    </Box>
+  );
+};
+
+// 2.2.c - Area Chart that Connects Nulls
+export const AreaChartConnectNulls: StoryFn<typeof ChartArea> = () => {
+  return (
+    <Box width="100%" height="100%">
+      <Heading size="small">Area Chart that does not Connects Nulls :- </Heading>
+      <Box width="500px" height="200px">
+        <ChartAreaWrapper data={data}>
+          <ChartCartesianGrid strokeDasharray="3 3" />
```

Comment:
Are the strokes of the cartesian grid not internally defined on figma? is it expected for these to be changeable?